src/pipeline-resolver.cpp

```
@@ -1,50 +1,417 @@
+
+    ~TaskBatch() { wait(); }
+
+    void submit(void (*func)(void*), void* payload, void (*payloadDeleter)(void*))
+    {
+        auto handle = m_taskPool->submitTask(func, payload, payloadDeleter, nullptr, 0);
+        SLANG_RHI_ASSERT(handle);
+        m_handles.push_back(handle);
+    }
+
+    void wait()
+    {
```

Cppcheck (2.21.0)

[style] The function 'create' is never used.

(unusedFunction)